sdk/include/crt/xmmintrin.h

```diff
@@ -29,7 +29,7 @@
 #endif
 
 #if !defined _VCRT_BUILD && !defined _INC_MALLOC
```

Comment:
A guess:
```suggestion
#if !defined(_VCRT_BUILD) && (!defined(_INC_MALLOC) && !defined(_MALLOC_H))
```

Reply:
Why?

Reply:
To fix i386 builds.

Reply:
it doesn't help at all indeed, because nowhere in ourr codebase is this `_MALLOC_H` existing.